integration_tests/features/steps/fal_steps.py

```diff
@@ -98,7 +102,7 @@ def invoke_failing_fal_flow(context):
 
 @then("model named {model_name} is removed")
 def delete_model(context, model_name):
-    os.system(f"rm {context.base_dir}/other_models/{model_name}.sql")
+    shutil.rmtree(os.path.join(context.base_dir, "other_models"))
```

Comment:
I think you are deleting the whole directory instead of `model_name`?

Reply:
What if we add an after_each behave hook that empties the `other_models` directory? We could get rid of this `@then`.

Reply:
But you changed the behavior, you are deleting all contents of the directory, not the model file as it was done before this PR

Reply:
Is this intentional that this folder is committed? 

https://github.com/fal-ai/fal/tree/main/integration_tests/projects/001_flow_run_with_selectors/other_models